RealmSwift/Tests/ObjectCreationTests.swift

```diff
@@ -237,15 +237,74 @@ class ObjectCreationTests: TestCase {
     }
 
     func testCreateWithNestedObjects() {
+        let standalone = SwiftPrimaryStringObject(object: ["11", 11])
+        Realm().beginWrite()
+        let objectWithNestedObjects = Realm().create(SwiftLinkToPrimaryStringObject.self, value: ["11", ["11", 11], [standalone]])
+        Realm().commitWrite()
+
+        let stringObjects = Realm().objects(SwiftPrimaryStringObject)
+        XCTAssertEqual(stringObjects.count, 1)
+        let persistedObject = stringObjects.first!
+
+        XCTAssertNotEqual(standalone, persistedObject)
```

Comment:
comment here would be useful - we test that standalone object is copied into realm